build.sbt

```diff
@@ -3,17 +3,9 @@ import com.scalapenos.sbt.prompt._
 import Dependencies._
 import microsites.ExtraMdFileConfig
 
-ThisBuild / crossScalaVersions := Seq("2.12.13", "2.13.4", "3.0.0-RC1")
+ThisBuild / crossScalaVersions := Seq("2.12.13", "2.13.5", "3.0.0-RC1")
 Test / parallelExecution := false
 
```

Comment:
Let's add `scalacOptions += "-Wconf:any:wv"` to the build
it will help us use fine-grained `@nowarn`